### PR TITLE
Fixes to previous merge request

### PR DESCRIPTION
**Pull request by vcfxb**
I removed comments and re-formatted the code to match rustfmt.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -87,7 +89,7 @@ impl From<checked_command::Error> for Error {
 
 impl error::Error for Error {
     fn description(&self) -> &str {
-        "Process error"
+        "Process error"
```
Comment (vcfxb) [inline src/lib.rs]: Extra space seems to be in error.

File: src/lib.rs [hunk 2]
```diff
@@ -75,6 +76,7 @@ impl From<checked_command::Error> for Error {
                 ex,
                 match err {
                     Some(e) => Output {
+                        stdout: String::from_utf8_lossy(&e.stdout).to_string(),
```
Comment (vcfxb) [inline src/lib.rs]: Is this line extraneous?

**Review by vcfxb — COMMENTED** [review]
There are a few fixes that need to be made actually.

**Comment by vcfxb** [comment]
I fixed those items in the pull-request, this should all be good to go.

**Comment by vcfxb** [comment]
Nevermind actually. looks like these changes have been made already.

**Comment by otavio** [comment]
Hello :-D I ended amending the commit in #1 and it was merged in c7e87c68

**Pull request closed without merging**